.circleci/config.yml

```diff
@@ -25,7 +25,7 @@ executors:
         environment:
           RAILS_ENV: test
 
-      - image: cimg/mysql:5.7
+      - image: cimg/mysql:8.0
```

Comment:
We won't want to change this here since that would lead to us testing 8, when we're still running 5.7 in production thus meaning we've got not test coverage for our current production reality.  We should test *both* 5.7 and 8.0 until we are fully on 8.0, then come back and remove 5.7 from the matrix.  Take a look at how the `elasticsearch_version` and `ruby_version` params are set and extend this to a `mysql_version` matrix.  (This repo isn't a great example because we're currently only testing against one version of ruby and elasticsearch, but i14y and asis's configs will show you a matrix with additional ruby version.)

This will likely also require some discussion with someone with access to branch protection rule settings - we'll want to ensure that passing checks are *required* for 5.7, but optional for 8.0, up until we make the switch over to 8.0 at which point 8.0 will need to be required and 5.7 should be removed entirely.

Reply:
@lorawoodford  - Makes sense. Will check on this.